project/frontend/src/components/SGonksLists/ShortSGonksList/ShortSGonksRow/ShortSGonksRow.js

```diff
@@ -2,13 +2,12 @@ import React from "react";
 import classes from "./ShortSGonksRow.module.css";
 
 const ShortSGonksRow = (props) => {
+  console.log(props);
   const getDiffIndicator = () => {
-    const diff =
-      props.sgonk.currentPrice > props.sgonk.priceYesterday
-        ? "+"
-        : props.sgonk.currentPrice < props.sgonk.priceYesterday
-        ? "-"
-        : "=";
+    const priceDelta =
+      props.datapoints[props.datapoints.length - 1] -
+      props.datapoints[props.datapoints.length - 2];
+    const diff = priceDelta > 0 ? "+" : priceDelta < 0 ? "-" : "=";
```

Comment:
nit for readability: complex ternaries often make things harder to read. I'd suggest adding some brackets & newlines to help here, eg
```js
const diff = (priceDelta > 0) ? "+" 
           : (priceDelta < 0) ? "-" 
           : "=";
```

Reply:
Linter says no :(